modules/data.atmosphere/R/met2CF.csv.R

```diff
@@ -26,7 +26,7 @@
 ##'   lon = -80
 ##'   met2CF.csv(in.path,in.file,outfolder,format,lat,lon)
 ##' }
-met2CF.csv <- function(in.path, in.file, outfolder, format, lat=NULL, lon=NULL, nc_verbose = FALSE){
+met2CF.csv <- function(in.path, in.file, outfolder, format, lat=NULL, lon=NULL, nc_verbose = FALSE,...){
```

Comment:
what will get passed with the `...`? (also in the other `met2CF.*` functions here)? I don't see anything currently; would be useful to add info to documentation as `##' @param ... `